bindings/java/Cargo.toml

```diff
@@ -156,14 +156,16 @@ anyhow = "1.0.71"
 jni = "0.21.1"
 num_cpus = "1.15.0"
 once_cell = "1.19.0"
-opendal = { workspace = true, features = ["layers-blocking"] }
+opendal = { path = "../../core", features = ["layers-blocking"] }
 tokio = { version = "1.28.1", features = ["full"] }
 
 # This is not optimal. See also the Cargo issue:
 # https://github.com/rust-lang/cargo/issues/1197#issuecomment-1641086954
 [target.'cfg(unix)'.dependencies.opendal]
+path = "../../core"
 features = [
+  "layers-blocking",
+
```

Comment:
IIRC the features would merge and we don't need this change?

Reply:
Fixed.